internal/indexer/definitions/milkie.yaml

```diff
@@ -0,0 +1,58 @@
+---
+#id: Milkie
+name: Milkie
+identifier: milkie
+description: Milkie is a private torrent tracker for GENERAL.
+language: en-us
+urls:
+  - https://www.milkie.cc
+privacy: private
+protocol: torrent
+supports:
+  - irc
+  - rss
+source: custom
+settings:
+  - name: apikey
+    type: secret
+    label: Api key
+    help: "Go to your profile and copy your Api key, make sure to replace <+> by <%2B>"
+    regex: /([a-zA-Z0-9\+])

```

Comment:
This seem to break at least syntax highlight, but did it load fine when you tested? How does the key actually look?

The regex for settings isn't yet used, but could could be useful.

Reply:
I tested this one completely and it worked well yes.
The api key looks like this: Cpi1Xt9fKYGtTln+6ilkuzGsWiJ7cSHq, this one is a fake one but same patent as mine.
Regex not updated in other files for the time being

Reply:
I see, so it's thats only to replace it for an url encoded value. Thats fine with the regex, can update that later on.